tools/s3s3mirror/src/main/java/org/cobbzilla/s3s3mirror/CopyMaster.java

```diff
@@ -0,0 +1,24 @@
+package org.cobbzilla.s3s3mirror;
```

Comment:
These `java` files are copied from cobbzilla? Do we intend to improve readability, have them conform to our style guide, etc.? Keep them as is?